### PR TITLE
Remove need for authentication/authorization

### PR DESCRIPTION
**Pull request by philipstanislaus**

### CODE REVIEW

File: containers/Investment/JuniorRatio/index.tsx [hunk 1]
```diff
@@ -61,19 +61,19 @@ class JuniorRatio extends React.Component<Props, State> {
...
                         <NumberInput value={baseToDisplay(minJuniorRatio, 27)} precision={2}
                             onValueChange={({ value }) =>
                                 this.setState({ minJuniorRatio: displayToBase(value, 27) })}
                         />
                     </FormField>
                 </Box>
                 <Box align="start">
-                    <Button primary label="Set min TIN ratio" onClick={this.setMinJuniorRatio} />
+                    <Button primary label="Set min TIN/DROP ratio" onClick={this.setMinJuniorRatio} />
                 </Box>
```
Comment (ilinzweilin) [inline containers/Investment/JuniorRatio/index.tsx:77]: I think it should be: Set min TIN ratio 🙈

File: containers/Investment/Tranche/index.tsx [hunk 2]
```diff
@@ -66,7 +66,7 @@ class TrancheView extends React.Component<Props> {
           {isInvestor &&
             <Box margin={{ top: 'medium', bottom: 'large' }} >
               <Box gap="medium" align="start" margin={{ bottom: 'medium' }} >
-                <Heading level="4" margin="none">Invest / Redeem </Heading>
```
Comment (ilinzweilin) [inline containers/Investment/Tranche/index.tsx:69]: We agreed to use Invest as a user facing term. In the contracts we are using supply though

File: containers/Investment/JuniorRatio/index.tsx (hunk 1 above)
Comment (philipstanislaus) [reply containers/Investment/JuniorRatio/index.tsx:77]: Okay, reset :)

File: containers/Investment/Tranche/index.tsx (hunk 2 above)
Comment (philipstanislaus) [reply containers/Investment/Tranche/index.tsx:69]: Reverted. I used the terms that @denniswell had in his screenshots, but it seems they were outdated.

**Comment by philipstanislaus** [comment]
Made header publicly visible, please have another look @ilinzweilin

**Comment by philipstanislaus** [comment]
Done, please approve :)

**Comment by ilinzweilin** [comment]
🎉🎉🎉

**Pull request merged**